xapian-glib/xapian-utils.h

```diff
@@ -39,9 +39,9 @@ int             xapian_minor_version            (void);
 XAPIAN_GLIB_AVAILABLE_IN_1_0
 int             xapian_revision                 (void);
 XAPIAN_GLIB_AVAILABLE_IN_1_4
-char *          xapian_sortable_serialise       (double value);
+guchar *        xapian_sortable_serialise       (double value, gsize *len);
```

Comment:
Coding style: arguments after the first go on a new line, aligned with the first argument.